src/realmGeolOrogen.ttl

```diff
@@ -151,23 +157,51 @@ soreagor:Orogen rdf:type owl:Class ;
                              owl:onProperty sorelph:hasPlanetaryStructure ;
                              owl:someValuesFrom sorealt:ThrustSystem
                            ] ;
-           rdfs:label "orogen"@en .
-
+           dcterms:source <http://www.glossaryofgeology.org> ;
+           dcterms:contributor <http://linked.data.gov.au/def/gsq> ;                           
+           skos:definition "An orogen (orogenic system, orogenic zone, orogenic belt) is a region in Earth’s lithosphere where a mountain belt is created by tectonic processes involving deformation, regional metamorphism, and associated magmatism, usually caused by convergence (± accretion and/or collision) between two tectonic plates or major crustal blocks. (Alt obsolete term: geosyncline). (Modified from Neuendorf et al., 2011; Hoy, pers. comm.)"@en ;                      
+           skos:prefLabel "orogen"@en .
+
+###  http://sweetontology.net/realmGeolOrogen/OrogenicCollage
+soreagor:OrogenicCollage rdf:type owl:Class ;
+            dcterms:source <http://www.glossaryofgeology.org> ;
+            dcterms:source <https://doi.org/10.1146/annurev-earth-082517-010146> ;
+            rdfs:subClassOf soreag:GeologicProvince ;
```

Comment:
Is every OrogenicCollage also a GeologicProvince? Is this true @smrgeoinfo ?

Reply:
Again, a good question and the answer is similar to that posed for supercratons: An orogenic collage consists of a number of orogens (more than one) of different age, e.g., the Tasmanides of eastern Australia. Some may consider the Tasmanides (Tasman Orogenic Zone) to represent a single orogen, but most workers regard the Tasmanides as embracing three separate orogens ranging in age from Neoproterozoic to late Middle Triassic (if not to the Norian, i.e., middle Late Triassic—taking into account what the South Africans consider for their Cape Orogeny, and disregarding early Late Triassic dextral transtension in eastern Australia).

So, an orogenic collage consists of the number of provinces that constitute it. Again, over to Nick on how to deal with this.

Reply:
The problem is with the proposed definition of 'Geologic Province'-- it is extensional, i.e. list of things that are Provinces: "cratons/shields (in the continental realm), orogens, sedimentary basins, and tectonised/metamorphosed and/or mineralised regions, as well as large igneous provinces"  I think there needs to be an intentional definition-- i.e. what are the properties of a part of the Earth that make it a 'Geologic Province'?

My proposal would be something like:
A geologic province is a part of the Earth defined by a shared geologic history. 

ScopeNote:
A given part of the Earth might be included in various geologic provinces based on different temporal intervals in its history.  For example, western Arizona is part of the North American Craton province in the Early to Middle Paleozoic; it is part of the SW North America Megashear province during the middle Mesozoic; it is part of the SW North America transpressional linkage province in the Late Cretaceous; it is part of the low-angle subduction hanging wall province in the Early Tertiary; it is part of the crustal extension province in the Oligocene and Miocene, and part of the Eastern California Shear zone province in the Late Miocene and Pliocene.

Reply:
OK with me & fits USGS & GA, especially the former in their definition!!

Reply:
sounds like this can be resolved?

Reply:
Yes